### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by FeisEater**

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 class CoursesController < ApplicationController
   before_action :set_organization
-  before_action :set_course, only: [:show, :edit, :update, :refresh, :manage_deadlines, :save_deadlines, :enable, :disable, :manage_unlocks, :save_unlocks ]
+  before_action :set_course, except: [:index, :show_json, :new, :create, :help]
```
Comment (jamox) [inline app/controllers/courses_controller.rb]: Please sort the names :)

File: app/controllers/organizations_controller.rb [hunk 2]
```diff
@@ -4,11 +4,11 @@ class OrganizationsController < ApplicationController
   skip_authorization_check only: [:index, :show]
 
   def index
-    @organizations = Organization.accepted_organizations
+    @organizations = Organization.accepted_organizations.order('name')
```
Comment (jamox) [inline app/controllers/organizations_controller.rb]: maybe in lower case (see samples below)

File: app/views/course_templates/index.html.erb [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
     <tr>
       <th>Name</th>
       <th>Title</th>
       <th>Description</th>
-      <th>Material url</th>
-      <th>Source url</th>
-      <th>Expires at</th>
+      <th>Material&nbsp;url</th>
+      <th>Source&nbsp;url</th>
+      <th>Expires&nbsp;at</th>
```
Comment (jamox) [inline app/views/course_templates/index.html.erb]: not sure if we like the `&nbsp` i'd prefer just ` `

File: app/views/course_templates/index.html.erb [hunk 4]
```diff
@@ -19,11 +19,11 @@
         <td><%= course_template.name %></td>
         <td><%= course_template.title %></td>
         <td><%= course_template.description %></td>
-        <td><%= course_template.material_url %></td>
-        <td><%= course_template.source_url %></td>
+        <td><%= link_to course_template.material_url, course_template.material_url %></td>
+        <td><%= link_to course_template.source_url, course_template.source_url %></td>
```
Comment (jamox) [inline app/views/course_templates/index.html.erb]: Umm, link to git repo -- rather have it in more easily copypasteable form

File: app/views/course_templates/index.html.erb (hunk 3 above)
Comment (mpartel) [inline app/views/course_templates/index.html.erb]: Might be easier to add some CSS that makes all `<th>` not wrap. Not sure if that's good for *all* our tables though (I'd check especially the points tables when large).

File: app/views/course_templates/index.html.erb (hunk 4 above)
Comment (mpartel) [inline app/views/course_templates/index.html.erb]: Links are fairly easy to copy in most browsers, but I agree in that it's a bit silly to have a link that one almost never ought to left-click.

Comment (jamox) [inline app/views/course_templates/index.html.erb]: I also see people accidentally clicking the link while attempting to copy it.
Though this information is rarely used anyways.

**Comment by jruo** [comment]
Updated

**Comment by mpartel** [comment]
LGTM assuming all tests pass.

**Comment by FeisEater** [comment]
While pr was not merged, I added one small fix myself.

**Pull request merged**